src/test/scala/ch/epfl/bluebrain/nexus/tests/Tags.scala

```diff
@@ -0,0 +1,17 @@
+package ch.epfl.bluebrain.nexus.tests
+
+import org.scalatest.Tag
+
+trait Tags
+
+/**
+ * Scalatest tags
+ */
+object Tags extends Tags {
+
+  object IamTag extends Tag("Iam")
```

Comment:
I'd try to avoid the use of the IAM concept. It shouldn't exist anymore in the code.
Please use every specific sub-resource if you prefer (identities, acls, realms, permissions)

Reply:
It is just a way to run all tests on identities, acls, realms, permissions otherwise it is quite verbose to type them all.

It is not so bad that we continue to refer to it not as the old module but as the acronym ? Or yes ?